front/pages/w/[wId]/ds/[name]/settings.tsx

```diff
@@ -572,7 +601,14 @@ function ManagedDataSourceSettings({
   };
   githubAppUrl: string;
   canUpdatePermissions: boolean;
+  handleUpdate: (settings: {
```

Comment:
This type is not coherent with the type of handle update? that's a bit surprising?